CHANGELOG.md

```
@@ -12,4 +32,4 @@
 
 ## [0.0.1] – 2025-04-18
 - 초기 릴리즈
-    - `lineLog`, `timeLog` 등 기본 기능 제공  
+    - `lineLog`, `timeLog` 등 기본 기능 제공
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)